tox.ini

```diff
@@ -1,43 +1,33 @@
-# Note: The testing environemnt depends on version 0.1.2 of opaque_keys, which
```

Comment:
This comment should probably be moved into the `setup.py` and/or `requirements.txt` so we don't lose the context.